dvc/remote/ssh/__init__.py

```diff
@@ -47,9 +51,19 @@ class RemoteSSH(RemoteBASE):
     # We use conservative setting of 4 instead to not exhaust max sessions.
     CHECKSUM_JOBS = 4
 
+    DEFAULT_CACHE_TYPES = ["reflink, copy"]
+
     def __init__(self, repo, config):
         super(RemoteSSH, self).__init__(repo, config)
 
+        types = config.get(Config.SECTION_CACHE_TYPE, None)
```

Comment:
Seems like you've forgot to remove this too after generalizing it in RemoteBASE :slightly_smiling_face: